src/h3lib/lib/geoCoord.c

```diff
@@ -332,3 +395,56 @@ int64_t H3_EXPORT(numHexagons)(int res) {
                                    569707381193162L};
     return nums[res];
 }
+
+double area_triangle(GeoCoord a, GeoCoord b, GeoCoord c) {
+    // Surface area of a spherical triangle given by three lat/lng points
+    // input: radians
+    // ouput: unit sphere area
+
+    double A, B, C, S, T, E;
```

Comment:
Nit 1: For consistency in the codebase, these should be lowercase.
Nit 2: I generally prefer explicit names to math-symbolic ones unless there's a very strong convention otherwise.

Reply:
updated!